Automattic/sensei
# Add header on quiz template on Learning Mode

Resolves https://github.com/Automattic/sensei/issues/7069

## Proposed Changes
* Added header on quiz template
* As the header on lesson and quiz is exactly the same, the header is moved to a common pattern file and the templates refer to that header pattern when needed just as we do with other themes.
* The quiz title was overlapping the header and becoming sticky when scrolled, just removed the sticky style for now to keep the header as it should for this design. Title area should be made proper under the issue for title and navigation

## Testing Instructions


1. Create a course with lessons and quizzes
2. Make sure to turn learning mode on
3. Visit the lessons and the quizzes from the frontend
4. Make sure they look the same as before for lesson, and that the header looks the same on both lesson and quiz
5. Check using all variations of Course theme
6. Check using other top themes for sensei

![image](https://github.com/Automattic/sensei/assets/6820724/3e564621-20ae-4ac2-97b9-562f0450a8f8)

## Pre-Merge Checklist
